binding-library/java/release_notes.md

```diff
@@ -1 +0,0 @@
-### Release notes
```

Comment:
The release notes are empty at the time of PR. The intension here was to add all changes that will go in next minor version in the notes, and then move its content in GitHub release notes when a new extension is released, and empty up the files again so it's ready for next version notes. I have updated the release pipelines already that does not treat release notes files specially. I would be replacing both files with a single file containing change-log across all versions. The notes for GitHub release will be added manually.